src/pages/Components/Tags/design.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Design'
+order: 4
+---
+
+## Tags sketch file is composed of three sections :
+
+The sketch file includes other form elements. Here is the section you'll find the radio.
+
+### **👾 For developers :**
+
+This part is a specific section for developers, **for a better accessibility**. There are all tag states : default, hover, focus, active, disable, etc.
```

Comment:
This is a specific category meant to describe tag states like hover, focus, active, disabled. 
This is useful mainly for developer handoff.